### ISSUE TITLE
[WebToolsE2E][Aspire] The "Instrument" and "Description" are grayed out in the Aspire dashboard Metrics

### ISSUE THREAD

**Issue by Rita003** [dotnet/aspire#5335]
**REGRESSION INFO**: Work in Aspire 8.1.0

**INSTALL STEPS**
1. Clean machine: Win11 x64 23h2 ENU
2. Install SDK 8.0.401
3. Install Aspire 8.1.1-preview.1.24416.7
4. Apply feed

**REPRO STEPS**
1. Open CMD, create an Aspire Starter App using following commands:
```
dotnet new aspire-starter -o AspireStarterApp
cd AspireStarterApp/AspireStarterApp.AppHost
```
2. Dotnet run the project in CMD, and open URL in the browser.
`dotnet run`
3. Click the "Metrics" option on the Aspire dashboard left, and click on any instrument.

**ACTUAL**
The "Instrument" and "Description" are grayed out
![Screenshot 2024-08-19 142620](https://github.com/user-attachments/assets/58cecc89-beb8-4ae4-bb43-a772168571ea)

**EXPECTED**
![Screenshot 2024-08-19 142006](https://github.com/user-attachments/assets/24a91660-2661-494c-a424-97631c6c9e9b)

**Comment by drewnoakes** [comment]
@JamesNK any idea about this change?

**Comment by JamesNK** [comment]
Might be related to Adam changing the column headers.

Is it still a bug after clearing the cache and refreshing?

**Comment by Rita003** [comment]
@JamesNK This issue still repro after clearing the cache and refreshing.

**Comment by JamesNK** [comment]
I see it in dev environment. @adamint please take a look.

**Comment by adamint** [comment]
@JamesNK if this needs any action within the next two weeks, it needs to be reassigned. I will be OOF beginning tomorrow

**Comment by Rita003** [comment]
Verify this issue fixed on the latest Aspire 8.1.1-preview.1.24420.9
